packages/dom-helper/lib/prop.js

```diff
@@ -2,6 +2,9 @@ export function isAttrRemovalValue(value) {
   return value === null || value === undefined;
 }
 
+var getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
```

Comment:
Can you confirm this works on IE8 (which we still have to support)?

https://msdn.microsoft.com/library/dd548686%28v=vs.94%29.aspx

As per that article, getOwnPropertyDescroptor does work in IE8 for "DOM Objects", so it may be perfectly fine, I just want to confirm.